sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -664,7 +711,19 @@ def begin_recognize_custom_forms_from_url(self, model_id, form_url, **kwargs):
 
         if not model_id:
             raise ValueError("model_id cannot be None or empty.")
+        
+        pages = kwargs.pop("pages", None)
 
+        # FIXME: part of this code will be removed once autorest can handle diff mixin
+        # signatures across API versions
+        if pages:
+            if self._api_version == FormRecognizerApiVersion.V2_1_PREVIEW:
+                kwargs.update({"pages": pages})
+            else:
+                raise ValueError(
+                    "'pages' is only available for API version V2_1_PREVIEW and up"
+                )
+        
```

Comment:
qq: will `pages` support for other prebuilts (business card, invoice) be in another PR? I wasn't sure cause I saw the issue tagged.

Reply:
Oh I might have missed them, but at this point I'll open another PR after this one is merged in and address the missing pages support for the other prebuilts. I'll update the description on this PR to not close that issue.

Reply:
Done, updated the description to not close the issue so I can add the other missing pages support.